.github/workflows/build_platformIO.yml

```
@@ -2,18 +2,66 @@ name: PlatformIO build
 
 on:
   pull_request:
+    types: [opened, reopened]
     paths:
       - ".github/workflows/build_platformIO.yml"
+      - "library.json"
       - "examples/**"
-      - "!examples/old_backups/recipes/pingpair_maple/**"
-
+      - "!examples/old_backups/**"
+      - "!examples/rf24_ATTiny/**"
   push:
     paths:
       - ".github/workflows/build_platformIO.yml"
+      - "library.json"
       - "examples/**"
-      - "!examples/old_backups/recipes/pingpair_maple/**"
+      - "!examples/old_backups/**"
+      - "!examples/rf24_ATTiny/**"
+  release:
+    types: [published, edited]
```

only trigger this workflow for release events of type `created`
```suggestion
    types: [created]
```